front/src/utils/strings.ts

```diff
@@ -34,3 +36,11 @@ export function snakeToCamel(str: string) {
 }
 
 export const emptyStringRegex = /^\s+$/;
+
+export function joinTranslationList(translationList: string, item: string): string {
+  return `${translationList ? `${translationList}.` : ''}${item}`;
+}
+
+export function isParamNull(str: string | null): string {
```

Comment:
Here you add a space before the string, so the function should be rename

Reply:
yes, use something like geti18nKeyForNull() , return a key and use it in your component